interface/src/avatar/MyHead.h

```diff
@@ -23,8 +23,8 @@ class MyHead : public Head {
 
 private:
     // disallow copies of the Head, copy of owning Avatar is disallowed too
-    MyHead(const Head&);
-    MyHead& operator= (const MyHead&);
+    MyHead(const Head&) = delete;
```

Comment:
Because Head != MyHead.  Thus my question in the first comment as to whether this is a bug or typo in the code, I'm struggling to make sense of it.

Reply:
Oh, good point. I'll take a look at that later and see if I can figure it out.

Reply:
Trying to think about it... that is a non-copy constructor and thus deleting it has no effect, we're only focused on the compiler-generated objects which at this point are copy-constructor, move-constructor, assign-self and assign-move, right?

So at best that line is useless and is likely a typo.  I'm okay with replacing it with a Q_DISABLE_COPY and see what breaks, if nothing does we can consider it good.

Reply:
Yes, I think this is a mistake.

You can't copy a Head into a MyHead, because where would whatever is added in MyHead come from? Such a thing can't happen automatically, so forbidding it even if legal shouldn't do anything.

Also, the comment in Head.h is identical, suggesting that it was copy/pasted from there and left by mistake.

So I think just Q_DISABLE_COPY it.

Reply:
Okay.  Think I already ended up doing that in a later commit too.

Reply:
Yep, I noticed the commit just a second too late.